include/caf/leds.rst

```diff
@@ -69,54 +68,74 @@ To define PWM ports used by the application, complete the following steps:
 			ch2-pin = <7>;
 		};
 
-	Make sure to configure all PWM ports that are used by the application.
-	In particular, the number of PWM ports used must match the value set in :option:`CONFIG_CAF_LEDS_COUNT`.
+	Make sure to configure all PWM ports and channels that are used by the application.
```

Comment:
How do I know all ports and channels used by the application are configured? The removed sentence provided some hints, now it is unclear.